content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -72,213 +114,233 @@ Once the :guilabel:`Lot/Serial Number` is generated, click the blank field next
+   The error message states that the product(s) in stock have no lot/serial number. However, a
+   lot/serial number can be assigned to the product by making an :doc:`inventory adjustment
+   <../../warehouses_storage/inventory_management/count_products>`.
+
+.. _inventory/product_management/incoming-outgoing-products:
 
-Serial numbers can be assigned for both **incoming** and **outgoing** goods. For incoming goods,
-serial numbers are assigned directly on the purchase order form. For outgoing goods, serial numbers
-are assigned directly on the sales order form.
+Create serial numbers to incoming or outgoing products
+------------------------------------------------------
 
-Assign serial numbers to newly received products
-------------------------------------------------
+Serial numbers can be assigned to both incoming and outgoing goods. The receipt and delivery order
+forms mirror one another; the instructions below can be followed to assign serial numbers in either
+form.
 
-Assigning serial numbers to **incoming** goods can be done on the receipt, by clicking the
-:ref:`Detailed Operations smart button <inventory/product_management/detailed-operations>` or by
-clicking the :guilabel:`⦙≣ (bulleted list)` icon in the product line.
+- **Incoming goods:** Assign serial numbers directly on the **receipt**. Receipts can be accessed by
+  navigating to :menuselection:`Inventory app --> Operations --> Receipts`.
+- **Outgoing goods:** Assign serial numbers directly on the **delivery order**. Receipts can be
+  accessed by navigating to :menuselection:`Inventory app --> Operations --> Deliveries`.
 
-.. seealso::
-   :doc:`create_sn`
+.. important::
+   Before assigning serial numbers on receipts or delivery orders, be sure that the ability to
+   :ref:`create new serial numbers by operations type
+   <inventory/product_management/operation-type-setting>` is enabled.
 
-.. image:: serial_numbers/assign-serial-numbers.png
+Lots/serial number field
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Serial numbers can be entered directly into the :guilabel:`Serial Numbers` field on a receipt or
+delivery order.
+
+.. image:: serial_numbers/enter-in-field.png
    :align: center
-   :alt: Display the bulleted list icon and Detailed Operations smart buttons.
+   :alt: Select value for Serial Number field on receipt.
 
-.. warning::
-   Clicking :guilabel:`Validate` before assigning a serial number to received quantities will cause
-   a :guilabel:`User Error` pop-up to appear. The pop-up requires entry of a lot or serial number
-   for the ordered products. The :abbr:`RFQ (request for quotation)` **cannot** be validated without
-   a serial number being assigned.
+.. important::
+   To make the :guilabel:`Serial Numbers` field visible on a receipt or delivery order, click the
+   :icon:`oi-settings-adjust` :guilabel:`(Adjust Settings)` icon, and in the drop-down menu, tick
+   the :guilabel:`Serial Numbers` checkbox.
```

Comment:
this is a **Tip** not **Important** block (adds some color variation in the doc too which looks nice/inviting)